databuilder/dsl.py

```diff
@@ -51,7 +51,7 @@
 
 class Dataset:
     """
-    Represents the dataset of patients in the defined study.
+    Represents the dataset of patients in the defined dataset definition.
```

Comment:
I'm not sure that this really means anything any more. TBF it wasn't a great comment before. Maybe just delete it? This module is going away soon anyway.